unit5/.cs50.yaml

```diff
@@ -0,0 +1,6 @@
+check50:
+  files: &check50_files
+    - !include "*"
```

Comment:
Minor thing, but including everything is the default so you don't need to specify `files` at all. This can just become:

```yaml
check50: true
submit50: true
```

This is what the config for the final project looks like as well.